<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Video</class>
 <widget class="QDialog" name="Video">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(154, 153, 150);</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>110</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: transparent; border: none; }
QPushButton:hover { background-color: rgba(255, 255, 255, 50%); }
QPushButton:pressed { background-color: rgba(255, 255, 255, 70%); }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="videoListWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>600</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backwordButton">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>520</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: transparent; border: none; }
QPushButton:hover { background-color: rgba(255, 255, 255, 50%); }
QPushButton:pressed { background-color: rgba(255, 255, 255, 70%); }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>520</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: transparent; border: none; }
QPushButton:hover { background-color: rgba(255, 255, 255, 50%); }
QPushButton:pressed { background-color: rgba(255, 255, 255, 70%); }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>520</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: transparent; border: none; }
QPushButton:hover { background-color: rgba(255, 255, 255, 50%); }
QPushButton:pressed { background-color: rgba(255, 255, 255, 70%); }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="repeatButton">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>520</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: transparent; border: none; }
QPushButton:hover { background-color: rgba(255, 255, 255, 50%); }
QPushButton:pressed { background-color: rgba(255, 255, 255, 70%); }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="forwardButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>520</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: transparent; border: none; }
QPushButton:hover { background-color: rgba(255, 255, 255, 50%); }
QPushButton:pressed { background-color: rgba(255, 255, 255, 70%); }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
